### PR TITLE
Remove Controls.Picker.TextTransform binding

### PR DESCRIPTION
**Pull request by buyaa-n**
### Remove Controls.Picker.TextTransform binding### 

Implements #https://github.com/dotnet/maui/issues/2989, https://github.com/dotnet/maui/issues/2987, https://github.com/dotnet/maui/issues/2988

### Removal made ###

- Removes TextTransform property binding from Picker
- Removes TextTransform property binding from DatePicker
- Removes TextTransform property binding from TimePicker

### PR Checklist ###

- [x] Targets the correct branch 
- [x] Tests are passing (or failures are unrelated) - passed on windows/android locally
- [x] Avoids any changes not essential to the handler property

#### Does this PR touch anything that might affect accessibility?
- No

### CODE REVIEW

File: src/Controls/src/Core/Picker.cs [hunk 1]
```diff
@@ -11,14 +11,8 @@
 namespace Microsoft.Maui.Controls
 {
 	/// <include file="../../docs/Microsoft.Maui.Controls/Picker.xml" path="Type[@FullName='Microsoft.Maui.Controls.Picker']/Docs" />
-	public partial class Picker : View, IFontElement, ITextElement, ITextAlignmentElement, IElementConfiguration<Picker>
+	public partial class Picker : InputView, IFontElement, ITextAlignmentElement, IElementConfiguration<Picker>
```
Comment (buyaa-n) [inline src/Controls/src/Core/Picker.cs:14]: I think Picker is a input control, please let me know if that is wrong and i need to add the property to the interface instead

**Comment by buyaa-n** [comment]
@PureWeen @hartez the PR is now ready for review, please run the CI tests

**Review by mattleibow — COMMENTED** [review]
Thanks for getting stuck into this. I feel it is half there, but we need to make sure we are not adding features into Core that only Controls require.

The Text property can be deduced from Items[SelectedIndex] so is not needed. In the case where you need to update the behaviour of the actual text on screen, you can maybe try overriding the SelectedIndex mapper instead to set the platform control's text - effectively bypassing the Core implementation of the mapper.

**Comment by buyaa-n** [comment]
@PureWeen @mattleibow as per offline discussion removed `TextTransform` binding from Picker/DatePicker/TimePicker.  Seems this is it, but please let me know if anything else need to be removed/updated

**Review by mattleibow — APPROVED** [review]

**Pull request merged**